buffet/package.json

```diff
@@ -46,7 +46,10 @@
     "babel-core": "7.0.0-bridge.0"
   },
   "dependencies": {
-    "font-awesome": "^4.7.0",
+    "@fortawesome/fontawesome-svg-core": "^1.2.22",
```

Comment:
Why do we need all these icon libraries?

Reply:
When using the fontawesome css and fontawesome font as earlier, the user has to download the entire font and css even if he or she does not use any of them. 
When moving to these libraries, the clients who are using webpack, parcel or any of the other bundlers with tree shaking can have a bundle with only the icons we use without any extra baggage